File: python/exercício_34.py
print()
print("=-"*60)
print()
print("--------------empréstimos------------------")
print()
print("=-"*60)
print() 
renda=float(input("Digite a sua renda mensal: "))
empréstimo= float(input("Digite o valor do emprestimo que deseja: "))
parcelas= int(input("Quantas parcelas deseja solicitar: "))

parcelaspagas = renda * 0.70
parcelastotal = parcelaspagas * parcelas
limite= renda * 10

if empréstimo > limite:
    print("O valor desejado não pode ser aprovado")
if parcelastotal < parcelas:
    print("O valor desejado não pode ser aprovado")
if empréstimo <= limite:
    if parcelastotal > parcelas:
        print(f'O valor do emprestimo de {empréstimo} foi aprovado com {parcelas} parcelas')

File: java/exercício_34.java
import javax.swing.JOptionPane;

public class exercício_34 {
    public static void main(String args[]){
        
        float valor,salario;
        int anos;
        int meses;
        float prestacao;
        valor = Float.parseFloat(JOptionPane.showInputDialog("Digite o valor do emprestimo solicitado: "));
        salario = Float.parseFloat(JOptionPane.showInputDialog("Digite o salário mensal : "));
        anos = Integer.parseInt(JOptionPane.showInputDialog("Quantos anos para pagar: "));
        
        meses = anos * 12;
        prestacao = valor / meses;
           if (prestacao > salario * 0.3){
                JOptionPane.showMessageDialog(null,"Infelizmente você não pode obter o empréstimo");
           }else{
              JOptionPane.showMessageDialog(null,"Valor da prestação: R$ " + prestacao + " Empréstimo OK");
           }

    }
}
